newscoop/src/Newscoop/NewscoopBundle/Resources/translations/plugins_manager.en.yml

```diff
@@ -32,6 +32,8 @@
 'newscoop.plugins_manager.clipboard': 'Copy to clipboard: Ctrl+C (Cmd + C for Mac), Enter'
 'newscoop.plugins_manager.downloads': 'downloads'
 'newscoop.plugins_manager.favers': 'favers'
+'newscoop.plugins_manager.private_plugins_not_writable': 'Make sure that private plugins directory is writable by your webserver. Looks like it is not.'
```

Comment:
If we can expand a variable here, I suggest providing the path to help the user, for example:

 The directory $PRIVATE_PLUGINS_DIR is not writable by your web server.